lib/components/narrative/metro/metro-itinerary.tsx

```diff
@@ -0,0 +1,476 @@
+            // Reset the scroll. Refs would be the more
+            // appropriate way to do this, but they don't work
+            setTimeout(
+              () => document.querySelector('.metro-itin')?.scrollIntoView(),
+              10
+            )
+          }}
+        >
+          <ItineraryWrapper
+            aria-label={intl.formatMessage(
+              {
+                id: 'components.MetroUI.itineraryDescription'
+              },
+              {
+                routes: getItineraryRoutes(itinerary, intl),
+                time: formatDuration(itinerary.duration, intl)
+              }
+            )}
+            className={`itin-wrapper${mini ? '-small' : ''}`}
+          >
+            {itineraryHasAccessibilityScores(itinerary) && (
+              <AccessibilityRating
+                gradationMap={accessibilityScoreGradationMap}
+                score={getAccessibilityScoreForItinerary(itinerary)}
+              />
+            )}
+            {!mini && (
+              <ItineraryGrid className="itin-grid">
+                <Routes enableDot={enableDot}>
+                  {renderRouteBlocks(itinerary.legs)}
+                </Routes>
+                <PrimaryInfo>
+                  <FormattedDuration duration={itinerary.duration} />
+                </PrimaryInfo>
+                <Spacer />
+                <SecondaryInfo className={isFlexItinerary ? 'flex' : ''}>
+                  {isFlexItinerary ? (
+                    <FlexIndicator
+                      isCallAhead={isCallAhead}
+                      isContinuousDropoff={isContinuousDropoff}
+                      phoneNumber={phone}
+                      shrink={false}
+                      textOnly
+                    />
+                  ) : (
+                    firstTransitStop && (
+                      <FormattedMessage
+                        id="components.MetroUI.fromStop"
+                        values={{ stop: firstTransitStop }}
+                      />
+                    )
+                  )}
+                </SecondaryInfo>
+                <SecondaryInfo>
+                  {maxTotalFare === null || maxTotalFare < 0 ? (
+                    <FormattedMessage id="common.itineraryDescriptions.noTransitFareProvided" />
+                  ) : (
+                    <FormattedMessage
+                      id="components.ItinerarySummary.fareCost"
+                      values={{
+                        maxTotalFare: (
+                          <FormattedNumber
+                            currency={fareCurrency}
+                            currencyDisplay="narrowSymbol"
+                            // This isn't a "real" style prop
+                            // eslint-disable-next-line react/style-prop-object
+                            style="currency"
+                            value={maxTotalFare / 100}
+                          />
+                        ),
+                        minTotalFare: (
```

Comment:
The min fare here is not being formatted with the correct $ and cents. It works to change the value to a template literal with a $ but threw a typescript error. I think something needs to be updated in the getFare() function in the utils/state file